### PR TITLE
[EventHub] Support py311 and Bump uamqp Version

### PR DESCRIPTION
**Pull request by kashifkhan**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/setup.py [hunk 1]
```diff
@@ -62,14 +62,15 @@
...
+        'Programming Language :: Python :: 3.11',
         'License :: OSI Approved :: MIT License',
     ],
     python_requires=">=3.7",
     zip_safe=False,
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
         "azure-core<2.0.0,>=1.14.0",
-        "uamqp>=1.6.0,<2.0.0",
+        "uamqp>=1.6.3,<2.0.0",
```
[the remark this answers is not in the document]
Comment (swathipil) [reply sdk/eventhub/azure-eventhub/setup.py:73]: yeah we made changes to support 3.11

**Review by swathipil — APPROVED** [review]
other than 2 comments lgtm!

**Review by scbedd — APPROVED** [review]
I expect this will work

**Pull request merged**